targets/LinuxDeployDependencies.proj

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="ProcessJsonFiles" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="DeployDependencies.proj" />
```

Comment:
Made it as a temp solution for Linux (as we have no blessed :raised_hands: Assets task there). This file can just be dropped when we get a common task from the NuGet team.